src/Visualization/Python/RenderBBH.py

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import os
+
+import click
+import rich.columns
+
+
+def ah_vis(ah_xmf, render_view):
+    import paraview.simple as pv
+
+    Ah_xmf = pv.XDMFReader(registrationName=ah_xmf, FileNames=[ah_xmf])
+    Ah_xmf.PointArrayStatus = ["RicciScalar"]
+    ricciScalarLUT = pv.GetColorTransferFunction("RicciScalar")
+    ricciScalarPWF = pv.GetOpacityTransferFunction("RicciScalar")
+    Ah_xmf_Display = pv.Show(
+        Ah_xmf, render_view, "UnstructuredGridRepresentation"
+    )
+    Ah_xmf_Display.SetScalarBarVisibility(render_view, False)
+    render_view.Update()
+    transform1 = pv.Transform(registrationName="Transform1", Input=Ah_xmf)
+    transform1.Transform = "Transform"
+    transform1.Transform.Translate = [0.0, 0.0, 2.5]
+    transform1_display = pv.Show(
+        transform1, render_view, "UnstructuredGridRepresentation"
+    )
+    pv.Hide(Ah_xmf, render_view)
+    transform1_display.SetScalarBarVisibility(render_view, False)
+    render_view.Update()
+    pv.ColorBy(transform1_display, None)
+    pv.HideScalarBarIfNotNeeded(ricciScalarLUT, render_view)
+    transform1_display.AmbientColor = [0.0, 0.0, 0.0]
+    transform1_display.DiffuseColor = [0.0, 0.0, 0.0]
+    pv.Hide3DWidgets(proxy=transform1.Transform)
+
+
+def render_bbh(
+    output_dir,
+    volume_xmf,
+    aha_xmf,
+    ahb_xmf,
+    camera_angle,
+    color_map,
+    show_grid,
+    stride,
+):
+    """Generate Pictures from XMF files for BBH Visualizations
```

Comment:
Please add a section to the docs where you explain each argument. It can also be helpful to add type annotations (see other files like `Domain.py`) and defaults for the arguments.